src/Engine/Component/GeometryComponent.cpp

```diff
@@ -54,15 +54,17 @@ const Index* GeometryComponent::roIndexRead() const {
 TriangleMeshComponent::TriangleMeshComponent( const std::string& name,
                                               Entity* entity,
                                               const Ra::Core::Asset::GeometryData* data ) :
-    GeometryComponent( name, entity ), m_displayMesh( nullptr ) {
+    GeometryComponent( name, entity ),
+    m_displayMesh( nullptr ) {
```

Comment:
Not formatted according to the required clang-format. Use clang-format >= 9.0

Reply:
I cannot install clang-format version 9 on my laptop, so please do so and PR on my branch